### PR TITLE
Update

### PR DESCRIPTION
**Pull request by apasel422**

### CODE REVIEW

**Comment by FlashCat** [comment]
r? @cgaebel

(I've picked a reviewer for you, use r? to override)

File: src/lib.rs [hunk 1]
```diff
@@ -564,11 +564,8 @@ impl<K: Hash + Eq, V, S: BuildHasher> LinkedHashMap<K, V, S> {
     /// assert_eq!(None, keys.next());
     /// ```
     #[cfg_attr(feature = "clippy", allow(needless_lifetimes))] // false positive
```
Comment (dtolnay) [inline src/lib.rs]: this lint suppression can be removed

File: src/lib.rs [hunk 2]
```diff
@@ -589,11 +586,8 @@ impl<K: Hash + Eq, V, S: BuildHasher> LinkedHashMap<K, V, S> {
     /// assert_eq!(None, values.next());
     /// ```
     #[cfg_attr(feature = "clippy", allow(needless_lifetimes))] // false positive
```
Comment (dtolnay) [inline src/lib.rs]: this too

File: src/lib.rs (hunk 1 above)
Comment (apasel422) [inline src/lib.rs]: Done, thanks.

**Pull request merged**